tests/dummy/app/controllers/application.js

```diff
@@ -14,7 +14,7 @@ export default Controller.extend({
   },
 
   expandedItem: computed('currentRouteName', function() {
-    if (this.get('currentRouteName').startsWith('layout')) {
+    if (this.get('currentRouteName').substr(0, 6) === "layout") {
```

Comment:
Right you are, but could you please use single quotes in .js files. Double quotes are for html-ish code.